packages/errors/src/index.ts

```diff
@@ -25,7 +25,6 @@ export const isError = (error: unknown): error is Error => {
   // Walk the prototype tree until we find a matching object.
   while (error) {
     if (Object.prototype.toString.call(error) === '[object Error]') return true;
-    // eslint-disable-next-line no-param-reassign -- TODO: Fix eslint error following @vercel/style-guide migration
     error = Object.getPrototypeOf(error);
   }
```

Comment:
This block is the only path I can't figure out how to test. 

I tried using:

```js
const err = Object.assign(new Error(), {});
t.ok(isError(err));
```

But the `err` returns true for `error instanceof Error` and never enters the while loop.

I need to figure out how to define an object that has a nested Error prototype.

Reply:
Ahh I think I got it:

```js
class Foo {}
const err = new Error();
Object.setPrototypeOf(err, Foo.prototype);
```